completions/ssh-keygen

```diff
@@ -80,10 +81,17 @@ _ssh_keygen()
                         compopt -o filenames
                         COMPREPLY=($(compgen -c -- "${cur#*=}"))
                         ;;
-                    checkpoint=* | challenge=*)
+                    checkpoint=* | challenge=* | write-attestation=*)
                         cur=${cur#*=}
                         _filedir
                         ;;
+                    application=)
```

Comment:
`ssh-keygen -O application=s[TAB]` didn't work.

```suggestion
                    application=*([!:=]))
```

If we simply add `*`, it seems `ssh:` will be repeated as many times as the TABs, so `:` and `=` (which breaks the word through `COMP_WORDBREAKS`) are excluded.

Reply:
Thanks, added with `[^:=]` in b252360 as using `^` to negate is more common for these cases than `!` in the codebase.

Reply:
Thanks!